### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by pastorp3**
## Calculator Milestone 4: calculations

In this milestone I:
- Created logic for the calculator
-  Implemented logic on App.js
-  Added logic for operators

### CODE REVIEW

**Comment by pastorp3** [comment]
Just on linter us not working, Which is no-unused-vars because on the description of the milestone it only said import the calculate function but it doesn't say anything about implementing it in any part of App.js

**Review by trekab — COMMENTED** [review]
#### Invalid! :stop_sign:
Hi @pastorp3! Unfortunately, I have to invalidate your code review request because you have linter errors. ([Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)) Consider making use of of the ESLint disable comments, i.e: `eslint-ignore <rule-to-be-ignored>` since your current error is temporary.

_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @pastorp3  👋

**You've done pretty well here, using short and modular functions for operations. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  const n1 = Big(nOne);
+  const n2 = Big(nTwo);
+  if (operation === '-') total = n1 - n2;
+  else if (operation === '+') total = n1 + n2;
+  else if (operation === 'X') total = n1 * n2;
+  else if (operation === '÷') {
+    if (n2 === '0') total = 'Cant divide by 0';
+    else total = n1 / n2;
+  }
+
```
Comment (mikenath223) [inline src/logic/operate.js]: - [ ] Kindly replace the operators here with big.js math operators. Eg: `total = n1.plus(n2)`.

**Review by wathigo — APPROVED** [review]
### APPROVED :heavy_check_mark: 
Great implementing the suggested changes.

**You may proceed to the next milestone.**

**Happy Coding :computer: 
Simon Wathigo**

**Pull request merged**